hackathon/templates/hackathon/hackathon-view.html

```diff
@@ -0,0 +1,127 @@
+{% extends "base.html" %}
+
+{% load static %}
```

Comment:
I don't think you need the extra blank lines before or after

Reply:
I do this out of habit as it's a visual representation of different statements. Personal opinion of mine, but I've changed this to reflect now 😃